analyses/plot_gen_geo_distance.py

```diff
@@ -35,8 +35,8 @@ def strains_mapping(alignment, metadata):
                 if row.strain in diction:
                     diction[row.strain]['location'] = row.location
     mapping = {strains: attributes for strains, attributes in diction.items()
-     if attributes['location'] is not '' and attributes['location'] is not '?'
-     and attributes['location'].startswith('530')}
+               if attributes['location'] != '' and attributes['location'] != '?'

```

Comment:
String comparison should be with `==` and `!=`.

Reply:
@trvrb I majorly screwed up this git history. I'm sorry :( Can I just delete this branch and create a new PR?